llama_stack/cli/stack/remove.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import argparse
+from pathlib import Path
+import shutil
+
+from llama_stack.cli.subcommand import Subcommand
+
+
+class StackRemove(Subcommand):
+     """Remove the build stack"""
+
+     def __init__(self, subparsers: argparse._SubParsersAction):
+        super().__init__()
+        self.parser = subparsers.add_parser(
+            "rm",
+            prog="llama stack rm",
+            description="Remove the build stack",
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        )
+        self._add_arguments()
+        self.parser.set_defaults(func=self._remove_stack_command)
+
+     def _add_arguments(self):
+            self.parser.add_argument(
+            "name",
+            type=str,
+            help="Name of the stack to delete",
+      )
+
+     def _remove_stack_command(self, args: argparse.Namespace) -> None:
+            stack_dir = Path(args.name)
+            if stack_dir.exists():
+                shutil.rmtree(stack_dir)
+                print(f"Successfully deleted stack: {args.name}")
+            else:
+                print(f"Stack not found: {args.name}")
```

Comment:
Let's add some validation that this is a stack, otherwise this will remove any arbitrary path?

Reply:
I agree with your suggestion, to start with the stack list command, the approach will be list all directories in the path
`.llama/distributions`?

Reply:
FYI https://github.com/meta-llama/llama-stack/pull/1406

Reply:
![Screenshot 2025-05-13 at 12 48 19 PM](https://github.com/user-attachments/assets/2f45ce7c-b0b6-48d5-8078-da777fd6404c)
![Screenshot 2025-05-13 at 12 48 28 PM](https://github.com/user-attachments/assets/31ff9aaf-c1ad-44a0-8100-8d3b208365e6)
![Screenshot 2025-05-13 at 12 52 25 PM](https://github.com/user-attachments/assets/93d277c9-3964-4b1a-81b3-b1b19558a953)

Reply:
noice